.github/workflows/master-validation.yml

```diff
@@ -17,13 +17,21 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
         with:
+          fetch-depth: 0 # required to access tags
           submodules: 'true'
 
-      - name: Build devcontainer image
-        run: docker build --tag devcontainer:1.0 .devcontainer
+      - name: Build & run devcontainer image
+        # this always builds a new image from scratch rather than from the build-devcontainer-image workflow output
+        # so that we pick up the latest versions of everything
+        id: devcontainer
```

Comment:
Same question about leaving a comment here noting that if you change this you should also probably change the other usages of this pattern in this folder?